npm2deb/mapper.py

```diff
@@ -37,9 +38,17 @@ def get_debian_package(self, node_module):
         result['version'] = None
         result['suite'] = None
         result['repr'] = None
+        db_package = None
 
         if node_module in self.json:
             db_package = self.json[node_module]
+        else:
+            for pattern in self.json.keys():
+                if _fullmatch(pattern.replace('*', '.+'), node_module):
```

Comment:
It seems like you are trying to use the unix filename matching syntax. It would be better to use the [fnmatch](https://docs.python.org/3/library/fnmatch.html) module for it. 

So, that the `pattern.replace` and the `_fullmatch` can be replaced with [`fnmatch.fnmatch`](https://docs.python.org/3/library/fnmatch.html#fnmatch.fnmatch)

The code would look like this:

```python
if fnmatch.fnmatch(node_module, pattern):
    # do things
```

And `fnmatch` supports `*`, `?` as the respective wildcards and `[seq]` and `[!seq]` and sequence and inverse sequence match respectively.